applications/ironside/se/update/src/main.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <ironside/se/update.h>
+#include <stdio.h>
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_REGISTER(app, CONFIG_APP_LOG_LEVEL);
+
+int main(void)
+{
+	int err;
+	const struct ironside_se_update_blob *update =
+		(const struct ironside_se_update_blob *)CONFIG_UPDATE_BLOB_ADDRESS;
+
+	err = ironside_se_update(update);
```

Comment:
Should this try to sanity check the update memory region if it isn't empty? I know the rpc call to secdom will just eventually fail but that return code might not be as straight forward as 'first 16 bytes all 0xFF, no image programmed'

Reply:
Its printed quite clearly in the terminal output, I can see if it can be communicated more explicit.